### DISCUSSION TITLE
Achievements not representative of my GitHub activity

### DISCUSSION THREAD

**Discussion by jgalan** [community/community#40069] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I have opened, reviewed and merged 100s of PRs but in my profile achievements I only see 3 open PRs.

https://github.com/jgalan

**Comment by radar-blip** [comment, marked as answer]
If you are refering to your achievement badges, the x3 means you've reached the silver level of that achievement. 
Hovering over your badge shows at least 128

![pully](https://user-images.githubusercontent.com/39196767/203966350-1c26099a-5d15-4b9d-8ca1-d3ade354486c.png)


For more info: [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Reply by jgalan** [reply]
Perhaps it would be good to have small stars, instead of a number. It is clear then that this is a rank.

**Reply by radar-blip** [reply]
Thats an excellent idea actually. Maybe bronze, silver, and gold stars. 
You could always make a feature request to introduce that idea to the GitHub team.
